### PR TITLE
Repatriate to master

### PR DESCRIPTION
**Pull request by aidanhahn**
## Description
This PR contains a CI job to automatically repatriate changes pushed to release/v2.0 back to master. There is also a one line fix to the website update PR that fixes existing auth issues.

### CODE REVIEW

File: .github/workflows/repatriate-lts.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+jobs:
+  repatriate-to-lts:
+    runs-on: ubuntu-latest
+    name: repatriate-to-lts
+    env:
+      REF: ${{ github.ref }}
+      RUN: ${{ github.run_id }}
+      PUSH_ID: ${{ github.event.push_id }}
+      GH_TAIL_REF: ${{ github.event.before }}
+      GIT_API_KEY: ${{ secrets.GH_API_KEY }}
```
Comment (anodelman) [inline .github/workflows/repatriate-lts.yml:16]: Should be GH_GITHUB_API_KEY?

**Review by anodelman — CHANGES_REQUESTED** [review]

File: .github/workflows/repatriate-lts.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  repatriate-to-lts:
+    runs-on: ubuntu-latest
+    name: repatriate-to-lts
+    env:
+      REF: ${{ github.ref }}
+      RUN: ${{ github.run_id }}
+      PUSH_ID: ${{ github.event.push_id }}
+      GH_TAIL_REF: ${{ github.event.before }}
+      GIT_API_KEY: ${{ secrets.GH_GITHUB_API_KEY }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (anodelman) [inline .github/workflows/repatriate-lts.yml:17]: You don't need this, use GH_GITHUB_API_KEY.  token=api key.

File: .github/workflows/repatriate.yml [hunk 3]
```diff
@@ -2,35 +2,21 @@ name: repatriate-to-lts
...
     runs-on: ubuntu-latest
-    name: repatriate-to-lts
+    name: repatriate-to-master
     env:
       REF: ${{ github.ref }}
       RUN: ${{ github.run_id }}
       PUSH_ID: ${{ github.event.push_id }}
-      GH_TAIL_REF: ${{ github.event.before }}
-      GIT_API_KEY: ${{ secrets.GH_API_KEY }}
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (anodelman) [inline .github/workflows/repatriate.yml:15]: This should be GH_GITHUB_API_KEY.

**Review by kflynn — APPROVED** [review]
Hit it!

**Review by anodelman — APPROVED** [review]
Look at that token usage!

**Pull request merged**